app/src/sharedTest/java/org/oppia/android/app/player/state/StateFragmentTest.kt

```diff
@@ -1418,6 +1418,7 @@ class StateFragmentTest {
     return find { text in it.first }?.second
   }
 
+  // TODO(#1675): Add NetworkModule once data module is migrated off of Moshi.
```

Comment:
NB: not specifically needed in this PR, but it's needed in #1876 since that's after the work manager integration PR or another which incorrectly added the network module dependency here. #1876 removed that module.